lib/main_common.pm

```diff
@@ -2401,116 +2401,186 @@ sub load_virtualization_tests {
 }
 
 sub load_hypervisor_tests {
+    # Check if the HOST_HYPERVISOR variable is set to 'xen', 'kvm', or 'qemu'
     return unless (get_var('HOST_HYPERVISOR') =~ /xen|kvm|qemu/);
-    return unless get_var('VIRT_PART');
-    my $windows = check_var('VIRT_PART', 'windows');
 
-    # Install hypervisor via autoyast or manually
-    loadtest "autoyast/prepare_profile" if get_var "AUTOYAST_PREPARE_PROFILE";
-    load_boot_tests if check_var('VIRT_PART', 'install');
+    # Only proceed if the VIRT_TEST variable is set
+    return unless (get_var('VIRT_TEST'));
```

Comment:
Is this ```VIRT_TEST``` a new variable ?

Reply:
yes, that's a new parameter. 
To avoid mixing with other tests, use it for isolation.

Reply:
I was wondering you can reuse existing setting, like ```VIRTUALIZATION```.

Reply:
@RoyCai7  Would you please explain more explicitly what would you plan to use this var to isolate for?